examples/rl/simple_example.ipynb

```diff
@@ -1,357 +0,0 @@
-{
```

Comment:
This is an excellent example.
I think it would be great to keep it as an RL tutorial.